### PR TITLE
Added components, assets and removed tailwind test data

### PR DESCRIPTION
**Pull request by ikayz**
Design assets were added in a directory to use as a reference. The components directory was created. Dummy test data was removed from the index file.

### CODE REVIEW

**Comment by ikayz** [comment]
@yochannah

**Comment by ikayz** [comment]
@yochannah I've noticed the merge conflict

**Comment by ikayz** [comment]
@yochannah can you look at this PR.

File: README.md [hunk 1]
```diff
@@ -1,36 +1,91 @@
...
+
+<!-- CONTACT -->
+
+## Contact
+
+Code Is Science - [@codeisscience](https://twitter.com/codeisscience)
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## Acknowledgements
```
Comment (yochannah) [inline README.md:88]: Love this section!

File: README.md [hunk 2]
```diff
@@ -1,36 +1,91 @@
...
-## Available Scripts
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/codeisscience/journal-policy-tracker-frontend">
+    <img src="logo.png" alt="Logo">
+  </a>
 
-In the project directory, you can run:
+  <h1 style="text-align:center">Journal Policy Tracker Frontend</h1>
```
Comment (yochannah) [inline README.md:17]: No biggie for now, but in the readme let's try to use markdown unless we really need HTML to keep maintenance simpler.

**Review by yochannah — COMMENTED** [review]
@ikayz the readme is great - really clear how to get started. 

I think this is probably okay to go - just one Q before I merge: when I yarn install and yarn start, my view is very css-less. Is this right? 

image

File: README.md (hunk 1 above)
Comment (ikayz) [reply README.md:88]: Awesome

File: README.md (hunk 2 above)
Comment (ikayz) [reply README.md:17]: True, I'll sort out the HTML issue in the next commit

File: README.md [hunk 3]
```diff
@@ -1,36 +1,91 @@
...
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
+To run this project locally on your machine you need the following installed
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+- nodejs
+- npm or yarn
 
-### `yarn eject`
+### Installation
```
[the remark this answers is not in the document]
Comment (ikayz) [reply README.md:43]: > @ikayz the readme is great - really clear how to get started.
> 
> I think this is probably okay to go - just one Q before I merge: when I yarn install and yarn start, my view is very css-less. Is this right?
> 
> image

Yeah, the styling is yet to be added for that part.

**Pull request merged**